lib/kino/process.ex

```diff
@@ -421,15 +423,36 @@ defmodule Kino.Process do
     {func_result, sequence_diagram}
   end
 
-  defp generate_participant_entry(pid, idx) do
+  # TODO: use :proc_lib.get_label/1 once we require OTP 27
+  if Code.ensure_loaded?(:proc_lib) and function_exported?(:proc_lib, :get_label, 1) do
+    defp get_label(pid), do: :proc_lib.get_label(pid)
+  else
+    defp get_label(_pid), do: :undefined
+  end
+
+  defp generate_participant_entry(pid, idx, process_label) do
     try do
       {:registered_name, name} = process_info(pid, :registered_name)
       "participant #{idx} AS #{module_or_atom_to_string(name)};"
     rescue
-      _ -> "participant #{idx} AS #35;PID#{:erlang.pid_to_list(pid)};"
+      _ ->
+        case process_label do
+          :undefined ->
+            "participant #{idx} AS #35;PID#{:erlang.pid_to_list(pid)};"
+
+          process_label ->
+            "participant #{idx} AS #{format_for_mermaid_participant_alias(pid, process_label)};"
+        end
     end
   end
 
+  defp format_for_mermaid_participant_alias(pid, process_label) do
+    pid_text = :erlang.pid_to_list(pid) |> List.to_string()
+
+    participant_alias = "#{inspect(process_label)}<br/>#{pid_text}"
+    String.replace(participant_alias, "\"", "")
```

Comment:
I see, in that case I would do the replace on inspect specifically for clarity:

```suggestion
    label = process_label |> inspect() |> String.replace(~s{"}, "")
    "#{label}<br/>#{pid_text}"
```